lib/dateTime.js

```diff
@@ -1128,10 +1128,24 @@ export function formatDateTime(date, options) {
 
 }
 
-export function formatDateTimeFromTimestamp(timestamp, options) {
+function parseLocalDateTimeFromTimestamp(timestamp, options) {
 	const utcDate = new Date(timestamp);
 	const local = convertJsDateToLocalDateTime(utcDate);
 	if (!local) return formatDateTime(utcDate, options);
-	const localDate = new Date(local.year, local.month - 1, local.date, local.hours, local.minutes, local.seconds);
+	return new Date(local.year, local.month - 1, local.date, local.hours, local.minutes, local.seconds);
+}
+
+export function formatDateTimeFromTimestamp(timestamp, options) {
+	const localDate = parseLocalDateTimeFromTimestamp(timestamp, options);
```

Comment:
Since this could return utc or local date, would probably just want to call this `const date`

Reply:
Updated